Kudu.Core/Deployment/DeploymentManager.cs

```diff
@@ -836,7 +837,10 @@ private string GetDeploymentManifestPath(string id)
 
         private string GetLogPath(string id, bool ensureDirectory = true)
         {
-            return Path.Combine(GetRoot(id, ensureDirectory), LogFile);
+            var logPath = Path.Combine(GetRoot(id, ensureDirectory), XmlLogFile);
+            return FileSystemHelpers.FileExists(logPath)
+                ? logPath
+                : Path.Combine(GetRoot(id, ensureDirectory), TextLogFile);
```

Comment:
We'll always use Text going forward. Xml is to support existing sites that already have the old format. But yes, comment would clarify.